app.go

```diff
@@ -34,6 +34,12 @@ func NewApp() *cli.App {
 			&cli.BoolFlag{
 				Name: "envauth", Usage: "set to enable authorization from the environment (not all commands support this)",
 			},
+			&cli.StringFlag{
+				Name: "logdir", Usage: "set to a path to directory containing log files belonging to single mmr to use that data instead of remote logs",
+			},
+			&cli.StringFlag{
+				Name: "logfile", Usage: "set to a path to file or '-' for stdin to use specific mmr log instead of remote logs",
```

Comment:
Its minor, but is there scope for confusion --loglevel and --logdir ?

Reply:
yeh, definitely - happy to change the name of the option to something more sensible